cmd/captplanet/setup.go

```diff
@@ -58,6 +58,9 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 	if !setupCfg.Overwrite && err == nil {
 		fmt.Println("A captplanet configuration already exists. Rerun with --overwrite")
 		return nil
+	} else if setupCfg.Overwrite && err == nil {
+		fmt.Println("overwriting existing captplanet config")
+		os.RemoveAll(setupCfg.BasePath)
```

Comment:
Are we sure nothing is reading from the files when we do it? Otherwise it will fail on Windows

Reply:
The only thing I could think of is if they were running `captplanet run` at the same time as they tried to do `captplanet setup`. Which I don't think is a case we need to worry about

Reply:
👍 LGTM